Igniting sustainer before booster causes incomplete CalcMap creation and null pointer exception

ORK to replicate: 
[TitanIII.zip](https://github.com/openrocket/openrocket/files/8155419/TitanIII.zip)

Go to the Simulation tab and run the simulation

<details>
  <summary>Bug Report (click this to show report)</summary>

---------- Bug report ----------

Please include a description about what actions you were performing when the exception occurred:

Running simulation with motors in booster pods

1.Motors in booster pods C6-0

2.Motor in main C6-5

3.

Include your email address (optional; it helps if we can contact you in case we need additional information):

sj_h1@live.com

(Do not modify anything below this line.)

---------- Exception stack trace ----------

net.sf.openrocket.gui.main.SwingExceptionHandler$InternalException: An exception occurred during the simulation at net.sf.openrocket.gui.main.SwingExceptionHandler.handleErrorCondition(SwingExceptionHandler.java:120) at net.sf.openrocket.gui.simulation.SimulationRunDialog$InteractiveSimulationWorker.simulationInterrupted(SimulationRunDialog.java:414) at net.sf.openrocket.gui.simulation.SimulationWorker.done(SimulationWorker.java:98) at java.desktop/javax.swing.SwingWorker$5.run(SwingWorker.java:750) at java.desktop/javax.swing.SwingWorker$DoSubmitAccumulativeRunnable.run(SwingWorker.java:847) at java.desktop/sun.swing.AccumulativeRunnable.run(AccumulativeRunnable.java:112) at java.desktop/javax.swing.SwingWorker$DoSubmitAccumulativeRunnable.actionPerformed(SwingWorker.java:857) at java.desktop/javax.swing.Timer.fireActionPerformed(Timer.java:317) at java.desktop/javax.swing.Timer$DoPostEvent.run(Timer.java:249) at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313) at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770) at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721) at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715) at java.base/java.security.AccessController.doPrivileged(Native Method) at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85) at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740) at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203) at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124) at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:117) at java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:190) at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:235) at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:233) at java.base/java.security.AccessController.doPrivileged(Native Method) at java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:233) at java.desktop/java.awt.Dialog.show(Dialog.java:1070) at java.desktop/java.awt.Component.show(Component.java:1716) at java.desktop/java.awt.Component.setVisible(Component.java:1663) at java.desktop/java.awt.Window.setVisible(Window.java:1031) at java.desktop/java.awt.Dialog.setVisible(Dialog.java:1005) at net.sf.openrocket.gui.main.SimulationPanel$3.actionPerformed(SimulationPanel.java:177) at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967) at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308) at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405) at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262) at java.desktop/javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:279) at java.desktop/java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:297) at java.desktop/java.awt.Component.processMouseEvent(Component.java:6635) at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342) at java.desktop/java.awt.Component.processEvent(Component.java:6400) at java.desktop/java.awt.Container.processEvent(Container.java:2263) at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5011) at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321) at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843) at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918) at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547) at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488) at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307) at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2772) at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843) at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772) at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721) at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715) at java.base/java.security.AccessController.doPrivileged(Native Method) at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85) at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95) at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745) at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743) at java.base/java.security.AccessController.doPrivileged(Native Method) at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85) at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742) at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203) at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124) at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113) at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109) at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101) at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90) Caused by: java.lang.NullPointerException

---------- Thread information ----------

Thread[AWT-EventQueue-0,6,main]

---------- System information ----------

OpenRocket version: 22.02.beta.01

OpenRocket source: default

OpenRocket location: C:\Program Files\OpenRocket\.\OpenRocket.jar

JOGL version: 2.4.0-rc-20210111

Current default locale: en_US

System properties:

awt.toolkit=sun.awt.windows.WToolkit

exe4j.consoleCodepage=cp0

exe4j.isInstall4j=true

exe4j.launchName=C:\Program Files\OpenRocket\OpenRocket.exe

exe4j.moduleName=C:\Program Files\OpenRocket\OpenRocket.exe

exe4j.semaphoreName=Local\c:_program_files_openrocket_openrocket.exe0

exe4j.tempDir=

exe4j.unextractedPosition=0

file.encoding=Cp1252

file.separator=\

install4j.appDir=C:\Program Files\OpenRocket\

install4j.exeDir=C:\Program Files\OpenRocket\

install4j.launcherId=59

install4j.swt=false

java.awt.graphicsenv=sun.awt.Win32GraphicsEnvironment

java.awt.printerjob=sun.awt.windows.WPrinterJob

java.class.path=C:\Program Files\OpenRocket\.install4j\i4jruntime.jar;C:\Program Files\OpenRocket\.\OpenRocket.jar

java.class.version=55.0

java.home=c:\program files\openrocket\jre

java.io.tmpdir=C:\Users\sj_h1\AppData\Local\Temp\

java.library.path=c:\program files\openrocket\jre\bin;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\ANT\apache-ant-1.9.14-bin\bin;C:\Program Files\dotnet\;C:\Program Files\Git\cmd;C:\Users\sj_h1\AppData\Local\Programs\Python\Python39\Scripts\;C:\Users\sj_h1\AppData\Local\Programs\Python\Python39\;C:\Users\sj_h1\AppData\Local\Microsoft\WindowsApps;;C:\Users\sj_h1\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\sj_h1\.dotnet\tools

java.runtime.name=OpenJDK Runtime Environment

java.runtime.version=11.0.14.1+1-LTS

java.specification.name=Java Platform API Specification

java.specification.vendor=Oracle Corporation

java.specification.version=11

java.util.Arrays.useLegacyMergeSort=true

java.vendor=BellSoft

java.vendor.url=https://bell-sw.com/

java.vendor.url.bug=https://bell-sw.com/support

java.version=11.0.14.1

java.version.date=2022-02-08

java.vm.compressedOopsMode=Zero based

java.vm.info=mixed mode

java.vm.name=OpenJDK 64-Bit Server VM

java.vm.specification.name=Java Virtual Machine Specification

java.vm.specification.vendor=Oracle Corporation

java.vm.specification.version=11

java.vm.vendor=BellSoft

java.vm.version=11.0.14.1+1-LTS

jdk.debug=release

jnlp.jogamp.tmp.cache.root=jln15077748752382825805

line.separator=\u000d\u000a

os.arch=amd64

os.name=Windows 10

os.version=10.0

path.separator=;

sun.arch.data.model=64

sun.awt.enableExtraMouseButtons=true

sun.awt.exception.handler=net.sf.openrocket.gui.main.SwingExceptionHandler$AwtHandler

sun.boot.library.path=c:\program files\openrocket\jre\bin

sun.cpu.endian=little

sun.cpu.isalist=amd64

sun.desktop=windows

sun.io.unicode.encoding=UnicodeLittle

sun.java.command=C:\Program Files\OpenRocket\OpenRocket.exe

sun.jnu.encoding=Cp1252

sun.management.compiler=HotSpot 64-Bit Tiered Compilers

sun.os.patch.level=

user.country=US

user.dir=C:\Program Files\OpenRocket

user.home=C:\Users\sj_h1

user.language=en

user.name=sj_h1

user.script=

user.timezone=America/Chicago

user.variant=

---------- Error log ----------

23113 0.000 USER (-) ===== 393 USER lines removed =====

23113 2588.952 USER (DoubleModel.java:338) SliderModel setValue called for DoubleModel[RocketFigure:Rotation] newValue=49 scaledValue=0.3078760800517997 converted=0.3141592653589793

23116 2588.977 USER (DoubleModel.java:338) SliderModel setValue called for DoubleModel[RocketFigure:Rotation] newValue=8 scaledValue=0.05026548245743669 converted=0.05235987755982989

23119 2589.006 USER (DoubleModel.java:338) SliderModel setValue called for DoubleModel[RocketFigure:Rotation] newValue=0 scaledValue=0.0 converted=0.0

23832 0.000 WARN (-) ===== 757 WARN lines removed =====

23832 2658.652 WARN (DoubleModel.java:950) DoubleModel[constant=1.0] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@2b8ccd29]

23833 2658.652 WARN (DoubleModel.java:950) DoubleModel[constant=0.2] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@2828b307]

23834 2658.652 WARN (DoubleModel.java:950) DoubleModel[constant=0.2] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@1276c1d8]

23835 2658.652 WARN (DoubleModel.java:950) DoubleModel[constant=0.2] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@543595e4]

23836 2661.681 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

23837 2661.681 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

23838 2661.681 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

23840 2661.686 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

23841 2661.686 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

23842 2661.686 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

23859 2887.975 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

23860 2887.975 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

23861 2887.975 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

23862 2887.978 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

23863 2887.978 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

23865 2887.981 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

23879 2905.304 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

23880 2905.305 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

23881 2905.305 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

23883 2905.310 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

23884 2905.310 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

23885 2905.310 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

31144 2964.712 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:OverrideMass] while still having listeners [javax.swing.JSlider$ModelListener@265f682f, javax.swing.plaf.basic.BasicSliderUI$Handler@51c7a59a]

31151 2964.712 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:OverrideCGX] while still having listeners [javax.swing.JSlider$ModelListener@6b01e7cd, javax.swing.plaf.basic.BasicSliderUI$Handler@4d8daf34]

31158 2964.712 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:OverrideCD] while still having listeners [javax.swing.JSlider$ModelListener@49d122b8, javax.swing.plaf.basic.BasicSliderUI$Handler@20b706cf]

31171 2964.712 WARN (DoubleModel.java:934) Invalidating DoubleModel[AppearanceBuilder:Shine] while still having listeners [javax.swing.JSlider$ModelListener@753fc288, javax.swing.plaf.basic.BasicSliderUI$Handler@3f2365f1]

31184 2964.713 WARN (DoubleModel.java:934) Invalidating DoubleModel[AppearanceBuilder:Rotation] while still having listeners [javax.swing.JSlider$ModelListener@25a18942, javax.swing.plaf.basic.BasicSliderUI$Handler@2d90655f]

31191 2964.713 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:OuterRadius] while still having listeners [javax.swing.JSlider$ModelListener@147b5c80, javax.swing.plaf.basic.BasicSliderUI$Handler@607f584e, net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@5108a24c]

31198 2964.713 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:InnerRadius] while still having listeners [javax.swing.JSlider$ModelListener@659a0a65, javax.swing.plaf.basic.BasicSliderUI$Handler@651d2f2b]

31205 2964.713 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:Thickness] while still having listeners [javax.swing.JSlider$ModelListener@61d4549, javax.swing.plaf.basic.BasicSliderUI$Handler@44f92849]

31212 2964.713 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:Length] while still having listeners [javax.swing.JSlider$ModelListener@5d9bb7f7, javax.swing.plaf.basic.BasicSliderUI$Handler@52764162]

31219 2964.713 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:AxialOffset] while still having listeners [javax.swing.JSlider$ModelListener@767fa628, javax.swing.plaf.basic.BasicSliderUI$Handler@2a7fb3df]

31226 2964.713 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:MotorOverhang] while still having listeners [javax.swing.JSlider$ModelListener@592eb1a2, javax.swing.plaf.basic.BasicSliderUI$Handler@2aaba815]

31250 2964.713 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:ClusterScale] while still having listeners [javax.swing.JSlider$ModelListener@2863751e, javax.swing.plaf.basic.BasicSliderUI$Handler@8a79174]

31257 2964.713 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:ClusterRotation] while still having listeners [javax.swing.JSlider$ModelListener@4e84bdd0, javax.swing.plaf.basic.BasicSliderUI$Handler@2d874ee9]

31264 2964.713 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:RadialPosition] while still having listeners [javax.swing.JSlider$ModelListener@675b5f5d, javax.swing.plaf.basic.BasicSliderUI$Handler@35e7144f]

31271 2964.713 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:RadialDirection] while still having listeners [javax.swing.JSlider$ModelListener@675486f9, javax.swing.plaf.basic.BasicSliderUI$Handler@33f11028]

31419 2995.900 WARN (DoubleModel.java:950) DoubleModel[constant=1.0] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@6c2144d3]

31420 2995.900 WARN (DoubleModel.java:950) DoubleModel[constant=1.0] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@45397dc6]

31421 2995.900 WARN (DoubleModel.java:950) DoubleModel[constant=3.141592653589793] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@31b607f8]

31422 2995.900 WARN (DoubleModel.java:950) DoubleModel[constant=1.0] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@6030b80f]

31423 2995.900 WARN (DoubleModel.java:950) DoubleModel[constant=1.0] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@7bb3b605]

31424 2995.900 WARN (DoubleModel.java:950) DoubleModel[constant=4.0] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@1892c6a7]

31425 2995.900 WARN (DoubleModel.java:950) DoubleModel[constant=0.2] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@5bf36df2]

31426 2995.900 WARN (DoubleModel.java:950) DoubleModel[constant=0.0] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@5108a24c]

31432 2995.974 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:OverrideMass] while still having listeners [javax.swing.JSlider$ModelListener@7de5a09b, javax.swing.plaf.basic.BasicSliderUI$Handler@780aaa44]

31439 2995.974 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:OverrideCGX] while still having listeners [javax.swing.JSlider$ModelListener@6e20bfbf, javax.swing.plaf.basic.BasicSliderUI$Handler@55f6f99c]

31446 2995.974 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:OverrideCD] while still having listeners [javax.swing.JSlider$ModelListener@74ef77b6, javax.swing.plaf.basic.BasicSliderUI$Handler@4056608]

31459 2995.974 WARN (DoubleModel.java:934) Invalidating DoubleModel[AppearanceBuilder:Shine] while still having listeners [javax.swing.JSlider$ModelListener@1482f317, javax.swing.plaf.basic.BasicSliderUI$Handler@3c83d7cd]

31472 2995.974 WARN (DoubleModel.java:934) Invalidating DoubleModel[AppearanceBuilder:Rotation] while still having listeners [javax.swing.JSlider$ModelListener@169b54f0, javax.swing.plaf.basic.BasicSliderUI$Handler@2a7fe7ca]

31479 2995.974 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:OuterRadius] while still having listeners [javax.swing.JSlider$ModelListener@212e0a20, javax.swing.plaf.basic.BasicSliderUI$Handler@44fa5a16, net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@2b2a9710]

31486 2995.974 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:InnerRadius] while still having listeners [javax.swing.JSlider$ModelListener@371138fd, javax.swing.plaf.basic.BasicSliderUI$Handler@64f45b29]

31493 2995.974 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:Thickness] while still having listeners [javax.swing.JSlider$ModelListener@48d7e971, javax.swing.plaf.basic.BasicSliderUI$Handler@539fe1f8]

31500 2995.975 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:Length] while still having listeners [javax.swing.JSlider$ModelListener@7d7b0dc, javax.swing.plaf.basic.BasicSliderUI$Handler@4bc16af2]

31507 2995.975 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:AxialOffset] while still having listeners [javax.swing.JSlider$ModelListener@7788dbd5, javax.swing.plaf.basic.BasicSliderUI$Handler@64c88f42]

31514 2995.975 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:MotorOverhang] while still having listeners [javax.swing.JSlider$ModelListener@5b4776b9, javax.swing.plaf.basic.BasicSliderUI$Handler@46ae2853]

31538 2995.975 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:ClusterScale] while still having listeners [javax.swing.JSlider$ModelListener@2a227dc6, javax.swing.plaf.basic.BasicSliderUI$Handler@7cfb29d0]

31545 2995.975 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:ClusterRotation] while still having listeners [javax.swing.JSlider$ModelListener@3036ad04, javax.swing.plaf.basic.BasicSliderUI$Handler@af52299]

31552 2995.975 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:RadialPosition] while still having listeners [javax.swing.JSlider$ModelListener@364623d9, javax.swing.plaf.basic.BasicSliderUI$Handler@525a0a66]

31559 2995.975 WARN (DoubleModel.java:934) Invalidating DoubleModel[InnerTube:RadialDirection] while still having listeners [javax.swing.JSlider$ModelListener@5fd04ef7, javax.swing.plaf.basic.BasicSliderUI$Handler@7ae2f23a]

31566 2999.250 USER (BasicFrame.java:509) Save selected

31608 3007.280 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

31609 3007.280 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

31610 3007.280 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

31611 3007.282 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

31612 3007.282 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

31614 3007.285 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

31618 3017.367 WARN (DoubleModel.java:950) DoubleModel[constant=1.0] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@5702c8cc]

31619 3017.367 WARN (DoubleModel.java:950) DoubleModel[constant=1.0] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@352e7332]

31620 3017.367 WARN (DoubleModel.java:950) DoubleModel[constant=1.0] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@485c8f8d]

31621 3017.367 WARN (DoubleModel.java:950) DoubleModel[constant=0.0] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@2b2a9710]

31622 3017.367 WARN (DoubleModel.java:950) DoubleModel[constant=0.2] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@4aa1ac75]

31623 3017.367 WARN (DoubleModel.java:950) DoubleModel[constant=4.0] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@11f84119]

31624 3017.368 WARN (DoubleModel.java:950) DoubleModel[constant=1.0] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@44cd5adc]

31625 3017.368 WARN (DoubleModel.java:950) DoubleModel[constant=3.141592653589793] being garbage-collected while having listeners [net.sf.openrocket.gui.adaptors.DoubleModel$ValueSliderModel@3339d68]

34218 3021.864 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

34219 3021.864 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

34220 3021.864 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

34221 3021.867 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

34222 3021.867 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

34224 3021.870 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

34234 0.000 INFO (-) ===== 168 INFO lines removed =====

34234 3026.379 INFO (BasicEventSimulationEngine.java:293) Queueing Ignition Event for: Side Booster Inner Tube/22adc96f / C6 - Armed @: 0.0

34235 3026.379 INFO (BasicEventSimulationEngine.java:293) Queueing Ignition Event for: Main Booster Inner Tube/2385c323 / C6 - Armed @: 0.0

34240 3026.379 INFO (BasicEventSimulationEngine.java:359) Igniting motor: Side Booster Inner Tube/22adc96f / C6 - Armed @0.0

34244 3026.379 INFO (BasicEventSimulationEngine.java:359) Igniting motor: Main Booster Inner Tube/2385c323 / C6 - Armed @0.0

34248 3026.379 INFO (BasicEventSimulationEngine.java:443) ==>> @ 0.00000; from Branch: Sustainer ---- Branching: Booster Set ----

41735 3026.983 INFO (BasicEventSimulationEngine.java:94) <<Finished simulating branch: Sustainer curTime:4.261341835069018 finTime:4.261341835069018

41736 3026.983 INFO (BasicEventSimulationEngine.java:88) >>Starting simulation of branch: Booster Set

41755 3042.082 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

41756 3042.082 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

41757 3042.082 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

41758 3042.084 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

41759 3042.084 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

41761 3042.087 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

41774 3060.296 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

41775 3060.296 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

41776 3060.296 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

41777 3060.298 USER (ThrustCurveMotorSelectionPanel.java:276) Selected table row -1, nothing selected

41778 3060.298 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

41780 3060.301 USER (ThrustCurveMotorSelectionPanel.java:271) Selected table row 793: ThrustCurveMotorSet[Estes C6, type=Single-use, count=2]

41785 3070.470 INFO (BasicEventSimulationEngine.java:88) >>Starting simulation of branch: Sustainer

41789 3070.470 INFO (BasicEventSimulationEngine.java:293) Queueing Ignition Event for: Side Booster Inner Tube/22adc96f / C6 - Armed @: 0.0

41790 3070.470 INFO (BasicEventSimulationEngine.java:293) Queueing Ignition Event for: Main Booster Inner Tube/2385c323 / C6 - Armed @: 0.0

41795 3070.471 INFO (BasicEventSimulationEngine.java:359) Igniting motor: Side Booster Inner Tube/22adc96f / C6 - Armed @0.0

41799 3070.471 INFO (BasicEventSimulationEngine.java:359) Igniting motor: Main Booster Inner Tube/2385c323 / C6 - Armed @0.0

41803 3070.471 INFO (BasicEventSimulationEngine.java:443) ==>> @ 0.00000; from Branch: Sustainer ---- Branching: Booster Set ----

49618 3071.022 INFO (BasicEventSimulationEngine.java:94) <<Finished simulating branch: Sustainer curTime:4.200186149204022 finTime:4.200186149204022

49619 3071.022 INFO (BasicEventSimulationEngine.java:88) >>Starting simulation of branch: Booster Set

49622 3071.066 ERROR (SwingExceptionHandler.java:119) An exception occurred during the simulation

> java.lang.NullPointerException

49623 3071.066 INFO (SwingExceptionHandler.java:142) Running in EDT, showing dialog

49624 3071.067 INFO (SwingExceptionHandler.java:201) Showing Exception dialog

49625 3073.254 USER (SwingExceptionHandler.java:220) User requested sending bug report

49630 3074.810 INFO (SimulationPanel.java:657) Running simulations took 4344 ms

49637 3094.273 INFO (BasicEventSimulationEngine.java:88) >>Starting simulation of branch: Sustainer

49641 3094.273 INFO (BasicEventSimulationEngine.java:293) Queueing Ignition Event for: Side Booster Inner Tube/22adc96f / C6 - Armed @: 0.0

49642 3094.273 INFO (BasicEventSimulationEngine.java:293) Queueing Ignition Event for: Main Booster Inner Tube/2385c323 / C6 - Armed @: 0.0

49647 3094.274 INFO (BasicEventSimulationEngine.java:359) Igniting motor: Side Booster Inner Tube/22adc96f / C6 - Armed @0.0

49651 3094.274 INFO (BasicEventSimulationEngine.java:359) Igniting motor: Main Booster Inner Tube/2385c323 / C6 - Armed @0.0

49655 3094.274 INFO (BasicEventSimulationEngine.java:443) ==>> @ 0.00000; from Branch: Sustainer ---- Branching: Booster Set ----

57472 3094.802 INFO (BasicEventSimulationEngine.java:94) <<Finished simulating branch: Sustainer curTime:4.011574572724732 finTime:4.011574572724732

57473 3094.802 INFO (BasicEventSimulationEngine.java:88) >>Starting simulation of branch: Booster Set

57476 3094.841 ERROR (SwingExceptionHandler.java:119) An exception occurred during the simulation

> java.lang.NullPointerException

57477 3094.841 INFO (SwingExceptionHandler.java:142) Running in EDT, showing dialog

57478 3094.841 INFO (SwingExceptionHandler.java:201) Showing Exception dialog

57479 3106.286 USER (SwingExceptionHandler.java:215) User chose not to fill bug report

57483 3106.291 INFO (SimulationPanel.java:178) Running simulations took 12019 ms

57487 3113.185 INFO (BasicEventSimulationEngine.java:88) >>Starting simulation of branch: Sustainer

57491 3113.185 INFO (BasicEventSimulationEngine.java:293) Queueing Ignition Event for: Side Booster Inner Tube/22adc96f / C6 - Armed @: 0.0

57492 3113.185 INFO (BasicEventSimulationEngine.java:293) Queueing Ignition Event for: Main Booster Inner Tube/2385c323 / C6 - Armed @: 0.0

57497 3113.185 INFO (BasicEventSimulationEngine.java:359) Igniting motor: Side Booster Inner Tube/22adc96f / C6 - Armed @0.0

57501 3113.186 INFO (BasicEventSimulationEngine.java:359) Igniting motor: Main Booster Inner Tube/2385c323 / C6 - Armed @0.0

57505 3113.186 INFO (BasicEventSimulationEngine.java:443) ==>> @ 0.00000; from Branch: Sustainer ---- Branching: Booster Set ----

65258 3113.690 INFO (BasicEventSimulationEngine.java:94) <<Finished simulating branch: Sustainer curTime:4.162730376083737 finTime:4.162730376083737

65259 3113.690 INFO (BasicEventSimulationEngine.java:88) >>Starting simulation of branch: Booster Set

65262 3113.734 ERROR (SwingExceptionHandler.java:119) An exception occurred during the simulation

> java.lang.NullPointerException

65263 3113.734 INFO (SwingExceptionHandler.java:142) Running in EDT, showing dialog

65264 3113.734 INFO (SwingExceptionHandler.java:201) Showing Exception dialog

65265 3116.110 USER (SwingExceptionHandler.java:215) User chose not to fill bug report

65269 3116.115 INFO (SimulationPanel.java:178) Running simulations took 2931 ms

65273 3131.945 INFO (BasicEventSimulationEngine.java:88) >>Starting simulation of branch: Sustainer

65277 3131.945 INFO (BasicEventSimulationEngine.java:293) Queueing Ignition Event for: Side Booster Inner Tube/22adc96f / C6 - Armed @: 0.0

65278 3131.945 INFO (BasicEventSimulationEngine.java:293) Queueing Ignition Event for: Main Booster Inner Tube/2385c323 / C6 - Armed @: 0.0

65283 3131.945 INFO (BasicEventSimulationEngine.java:359) Igniting motor: Side Booster Inner Tube/22adc96f / C6 - Armed @0.0

65287 3131.945 INFO (BasicEventSimulationEngine.java:359) Igniting motor: Main Booster Inner Tube/2385c323 / C6 - Armed @0.0

65291 3131.945 INFO (BasicEventSimulationEngine.java:443) ==>> @ 0.00000; from Branch: Sustainer ---- Branching: Booster Set ----

65883 0.000 DEBUG (-) ===== 2776 DEBUG lines removed =====

65883 3131.985 DEBUG (SimulationRunDialog.java:236) Progressbar value 5

66813 3132.041 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 55.4%, estimate=8.588816197030148 correct=13.344000000000001 timestep=0.0025, recomputing k1 parameters

66944 3132.048 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 60.0%, estimate=0.07639255509014167 correct=0.030557022036053955 timestep=0.0025, recomputing k1 parameters

66949 3132.048 DEBUG (BasicEventSimulationEngine.java:409) detected Motor Burnout for motor C6@ 1.86 on stage 0: Sustainer

66980 3132.050 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 29.4%, estimate=3.5209180934856033 correct=2.4861727678639345 timestep=0.0035405002208581072, recomputing k1 parameters

66988 3132.050 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=4.741398899816757 correct=3.1609325998778384 timestep=0.00531075033128716, recomputing k1 parameters

66996 3132.051 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 112.4%, estimate=6.318305422722399 correct=13.42197704675909 timestep=0.0025, recomputing k1 parameters

67053 3132.055 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 32.3%, estimate=59.31895605348769 correct=40.18454093486779 timestep=0.0036904089653301735, recomputing k1 parameters

67061 3132.055 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=48.33626268602133 correct=32.22417512401422 timestep=0.005535613447995261, recomputing k1 parameters

67069 3132.055 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 69.7%, estimate=41.316241467295136 correct=70.09526016807136 timestep=0.003262856023910777, recomputing k1 parameters

67077 3132.056 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 30.5%, estimate=80.08409359214271 correct=104.52114687862405 timestep=0.0025, recomputing k1 parameters

67121 3132.058 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=165.11682112060063 correct=110.07788074706404 timestep=0.0040096664630539635, recomputing k1 parameters

67129 3132.059 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=123.9009323542089 correct=82.60062156947261 timestep=0.006014499694580945, recomputing k1 parameters

67137 3132.059 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 140.6%, estimate=97.44536703499178 correct=234.43405210812202 timestep=0.0025, recomputing k1 parameters

67174 3132.061 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=311.02824224720723 correct=207.35216149814414 timestep=0.0038404149193648988, recomputing k1 parameters

67182 3132.062 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=225.76171426271634 correct=150.50780950847755 timestep=0.005760622379047348, recomputing k1 parameters

67190 3132.062 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 130.4%, estimate=169.8959295361571 correct=391.4825175180269 timestep=0.0025, recomputing k1 parameters

67212 3132.063 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 28.3%, estimate=453.70156790542825 correct=325.17998477447816 timestep=0.0034880803643256107, recomputing k1 parameters

67220 3132.064 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=347.0396071971553 correct=231.35973813143195 timestep=0.005232120546488416, recomputing k1 parameters

67228 3132.064 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 109.3%, estimate=254.10815026232666 correct=531.8097896070911 timestep=0.0025, recomputing k1 parameters

67243 3132.065 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 16.6%, estimate=579.3923922879724 correct=483.23119989317485 timestep=0.0029974906029249524, recomputing k1 parameters

67251 3132.066 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=507.837630083529 correct=338.55842005567763 timestep=0.004496235904387429, recomputing k1 parameters

67259 3132.066 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 79.8%, estimate=363.9286339092771 correct=654.5235961670356 timestep=0.0025, recomputing k1 parameters

67274 3132.067 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=707.1997950393595 correct=471.4665300262397 timestep=0.00375, recomputing k1 parameters

67282 3132.067 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 24.1%, estimate=498.69645638547854 correct=618.836752710465 timestep=0.0030219790651645757, recomputing k1 parameters

67290 3132.068 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 20.9%, estimate=646.7569000128841 correct=781.7943248358387 timestep=0.0025, recomputing k1 parameters

67298 3132.069 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=810.2772892353562 correct=540.1848594902054 timestep=0.00375, recomputing k1 parameters

67306 3132.069 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 16.1%, estimate=569.3048431204203 correct=477.85033987258987 timestep=0.004467702507590499, recomputing k1 parameters

67314 3132.073 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 22.8%, estimate=507.8078970276166 correct=623.47200102651 timestep=0.0036388716914138052, recomputing k1 parameters

67322 3132.074 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=654.2558051672047 correct=436.17053677813647 timestep=0.005458307537120708, recomputing k1 parameters

67338 3132.075 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=468.39115767609087 correct=312.2607717840606 timestep=0.008795710795159915, recomputing k1 parameters

67346 3132.075 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=346.6199163739496 correct=231.07994424929194 timestep=0.013193566192739873, recomputing k1 parameters

67354 3132.076 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=267.6803036091015 correct=178.453535739405 timestep=0.01979034928910981, recomputing k1 parameters

67362 3132.076 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 33.3%, estimate=218.41571725409537 correct=145.6104781693958 timestep=0.029685523933664717, recomputing k1 parameters

67370 3132.077 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 1087.4%, estimate=190.61539291640733 correct=2263.407123418008 timestep=0.0025, recomputing k1 parameters

67455 3132.082 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 23.7%, estimate=2876.4660873471653 correct=2195.743624727922 timestep=0.0032750477502850434, recomputing k1 parameters

67463 3132.082 DEBUG (RK4SimulationStepper.java:190) Thrust estimate differs from correct value by 31.0%, estimate=2250.223366973489 correct=2947.835590258147 timestep=0.0025, recomputing k1 parameters

67621 3132.092 DEBUG (SimulationRunDialog.java:444) APOGEE, setting progress

67622 3132.092 DEBUG (SimulationRunDialog.java:424) Setting progress to 70 (real 70)

68449 3132.141 DEBUG (SimulationRunDialog.java:367) Method 3: alt=315.77369407982894 apogee=355.18267514641735

68450 3132.141 DEBUG (SimulationRunDialog.java:424) Setting progress to 73 (real 73)

68451 3132.141 DEBUG (SimulationRunDialog.java:236) Progressbar value 73

69208 3132.188 DEBUG (SimulationRunDialog.java:367) Method 3: alt=374.1104758126514 apogee=355.18267514641735

69209 3132.188 DEBUG (SimulationRunDialog.java:424) Setting progress to 73 (real 73)

69210 3132.188 DEBUG (SimulationRunDialog.java:236) Progressbar value 73

72450 3132.390 DEBUG (SimulationRunDialog.java:367) Method 3: alt=362.54069017244785 apogee=355.18267514641735

72451 3132.390 DEBUG (SimulationRunDialog.java:424) Setting progress to 73 (real 73)

72453 3132.390 DEBUG (SimulationRunDialog.java:236) Progressbar value 73

73338 0.000 VBOSE (-) ===== 69047 VBOSE lines removed =====

73338 3132.445 VBOSE (BasicEventSimulationEngine.java:265) Obtained event from queue: FlightEvent[type=ALTITUDE,time=4.223896636777925,source=Titan III,data=[259.10215739653046;259.52434956418284]]

73339 3132.445 VBOSE (BasicEventSimulationEngine.java:266) Remaining EventQueue = [FlightEvent[type=EJECTION_CHARGE,time=6.861793123367075,source=Sustainer,data=C6]]

73340 3132.445 VBOSE (BasicEventSimulationEngine.java:348) Handling event FlightEvent[type=ALTITUDE,time=4.223896636777925,source=Titan III,data=[259.10215739653046;259.52434956418284]]

73341 3132.445 VBOSE (BasicEventSimulationEngine.java:137) Taking simulation step at t=4.223896636777925 altitude 259.52434956418284

73342 3132.445 VBOSE (RK4SimulationStepper.java:169) Too small time step 1.9807495220107774E-8 (limiting factor 4), using 0.0025 instead.

73343 3132.445 VBOSE (RK4SimulationStepper.java:185) Thrust = 227818.7595358155

73344 3132.445 VBOSE (BasicEventSimulationEngine.java:263) HandleEvents: current branch = Sustainer

73345 3132.445 VBOSE (BasicEventSimulationEngine.java:265) Obtained event from queue: FlightEvent[type=ALTITUDE,time=4.226396636777926,source=Titan III,data=[259.52434956418284;260.35994683221867]]

73346 3132.445 VBOSE (BasicEventSimulationEngine.java:266) Remaining EventQueue = [FlightEvent[type=EJECTION_CHARGE,time=6.861793123367075,source=Sustainer,data=C6]]

73347 3132.445 VBOSE (BasicEventSimulationEngine.java:348) Handling event FlightEvent[type=ALTITUDE,time=4.226396636777926,source=Titan III,data=[259.52434956418284;260.35994683221867]]

73348 3132.445 VBOSE (BasicEventSimulationEngine.java:137) Taking simulation step at t=4.226396636777926 altitude 260.35994683221867

73349 3132.446 VBOSE (RK4SimulationStepper.java:169) Too small time step 1.0154322531611584E-8 (limiting factor 4), using 0.0025 instead.

73350 3132.446 VBOSE (RK4SimulationStepper.java:185) Thrust = 228300.8855810049

73351 3132.446 VBOSE (BasicEventSimulationEngine.java:263) HandleEvents: current branch = Sustainer

73352 3132.446 VBOSE (BasicEventSimulationEngine.java:265) Obtained event from queue: FlightEvent[type=ALTITUDE,time=4.228896636777926,source=Titan III,data=[260.35994683221867;255.79660400343886]]

73353 3132.446 VBOSE (BasicEventSimulationEngine.java:266) Remaining EventQueue = [FlightEvent[type=EJECTION_CHARGE,time=6.861793123367075,source=Sustainer,data=C6]]

73354 3132.446 VBOSE (BasicEventSimulationEngine.java:348) Handling event FlightEvent[type=ALTITUDE,time=4.228896636777926,source=Titan III,data=[260.35994683221867;255.79660400343886]]

73355 3132.446 VBOSE (BasicEventSimulationEngine.java:137) Taking simulation step at t=4.228896636777926 altitude 255.79660400343886

73356 3132.446 VBOSE (RK4SimulationStepper.java:169) Too small time step 3.36504880890629E-8 (limiting factor 5), using 0.0025 instead.

73357 3132.446 VBOSE (RK4SimulationStepper.java:185) Thrust = 228783.5212415788

73358 3132.446 VBOSE (BasicEventSimulationEngine.java:263) HandleEvents: current branch = Sustainer

73359 3132.446 VBOSE (BasicEventSimulationEngine.java:265) Obtained event from queue: FlightEvent[type=ALTITUDE,time=4.2313966367779265,source=Titan III,data=[255.79660400343886;256.24693350815687]]

73360 3132.446 VBOSE (BasicEventSimulationEngine.java:266) Remaining EventQueue = [FlightEvent[type=EJECTION_CHARGE,time=6.861793123367075,source=Sustainer,data=C6]]

73361 3132.446 VBOSE (BasicEventSimulationEngine.java:348) Handling event FlightEvent[type=ALTITUDE,time=4.2313966367779265,source=Titan III,data=[255.79660400343886;256.24693350815687]]

73362 3132.446 VBOSE (BasicEventSimulationEngine.java:137) Taking simulation step at t=4.2313966367779265 altitude 256.24693350815687

73363 3132.446 VBOSE (RK4SimulationStepper.java:169) Too small time step 6.281800306154865E-8 (limiting factor 5), using 0.0025 instead.

73364 3132.446 VBOSE (RK4SimulationStepper.java:185) Thrust = 229266.66651753738

73365 3132.447 VBOSE (BasicEventSimulationEngine.java:263) HandleEvents: current branch = Sustainer

73366 3132.447 VBOSE (BasicEventSimulationEngine.java:265) Obtained event from queue: FlightEvent[type=ALTITUDE,time=4.233896636777927,source=Titan III,data=[256.24693350815687;255.82151839054615]]

73367 3132.447 VBOSE (BasicEventSimulationEngine.java:266) Remaining EventQueue = [FlightEvent[type=EJECTION_CHARGE,time=6.861793123367075,source=Sustainer,data=C6]]

73368 3132.447 VBOSE (BasicEventSimulationEngine.java:348) Handling event FlightEvent[type=ALTITUDE,time=4.233896636777927,source=Titan III,data=[256.24693350815687;255.82151839054615]]

73369 3132.447 VBOSE (BasicEventSimulationEngine.java:137) Taking simulation step at t=4.233896636777927 altitude 255.82151839054615

73370 3132.447 VBOSE (RK4SimulationStepper.java:169) Too small time step 6.506573295577865E-9 (limiting factor 5), using 0.0025 instead.

73371 3132.447 VBOSE (RK4SimulationStepper.java:185) Thrust = 229750.32140888053

73372 3132.447 VBOSE (BasicEventSimulationEngine.java:263) HandleEvents: current branch = Sustainer

73373 3132.447 VBOSE (BasicEventSimulationEngine.java:265) Obtained event from queue: FlightEvent[type=ALTITUDE,time=4.236396636777927,source=Titan III,data=[255.82151839054615;250.62339470813004]]

73374 3132.447 VBOSE (BasicEventSimulationEngine.java:266) Remaining EventQueue = [FlightEvent[type=EJECTION_CHARGE,time=6.861793123367075,source=Sustainer,data=C6]]

73375 3132.447 VBOSE (BasicEventSimulationEngine.java:348) Handling event FlightEvent[type=ALTITUDE,time=4.236396636777927,source=Titan III,data=[255.82151839054615;250.62339470813004]]

73376 3132.447 VBOSE (BasicEventSimulationEngine.java:137) Taking simulation step at t=4.236396636777927 altitude 250.62339470813004

73377 3132.447 VBOSE (RK4SimulationStepper.java:169) Too small time step 1.9710227780856684E-8 (limiting factor 5), using 0.0025 instead.

73378 3132.447 VBOSE (RK4SimulationStepper.java:185) Thrust = 230234.48591560838

73379 3132.447 VBOSE (BasicEventSimulationEngine.java:263) HandleEvents: current branch = Sustainer

73380 3132.447 VBOSE (BasicEventSimulationEngine.java:265) Obtained event from queue: FlightEvent[type=ALTITUDE,time=4.238896636777928,source=Titan III,data=[250.62339470813004;256.1386482727]]

73381 3132.447 VBOSE (BasicEventSimulationEngine.java:266) Remaining EventQueue = [FlightEvent[type=EJECTION_CHARGE,time=6.861793123367075,source=Sustainer,data=C6]]

73382 3132.447 VBOSE (BasicEventSimulationEngine.java:348) Handling event FlightEvent[type=ALTITUDE,time=4.238896636777928,source=Titan III,data=[250.62339470813004;256.1386482727]]

73383 3132.447 VBOSE (BasicEventSimulationEngine.java:137) Taking simulation step at t=4.238896636777928 altitude 256.1386482727

73384 3132.448 VBOSE (RK4SimulationStepper.java:169) Too small time step 5.496564192398361E-11 (limiting factor 5), using 0.0025 instead.

73385 3132.448 VBOSE (RK4SimulationStepper.java:185) Thrust = 230719.1600377208

73386 3132.448 DEBUG (FlightData.java:247) Computed flight values: maxAltitude=396.9182820667803 maxVelocity=5.188603422566388E10 maxAcceleration=1.245550891024932E14 maxMachNumber=203638.53407852934 timeToApogee=2.791396636777916 flightTime=4.241396636777928 groundHitVelocity=NaN launchRodVelocity=23.067149710989064

73387 3132.448 INFO (BasicEventSimulationEngine.java:94) <<Finished simulating branch: Sustainer curTime:4.241396636777928 finTime:4.241396636777928

73388 3132.448 INFO (BasicEventSimulationEngine.java:88) >>Starting simulation of branch: Booster Set

73389 3132.448 VBOSE (BasicEventSimulationEngine.java:263) HandleEvents: current branch = Booster Set

73390 3132.448 VBOSE (BasicEventSimulationEngine.java:137) Taking simulation step at t=0.0 altitude 0.0

73391 3132.482 ERROR (SwingExceptionHandler.java:119) An exception occurred during the simulation

> java.lang.NullPointerException

73392 3132.482 INFO (SwingExceptionHandler.java:142) Running in EDT, showing dialog

73393 3132.482 INFO (SwingExceptionHandler.java:201) Showing Exception dialog

73394 3135.317 USER (SwingExceptionHandler.java:220) User requested sending bug report

73395 3135.318 DEBUG (JarUtil.java:40) Found jar file using codeSource

---------- End of bug report ----------

</details>

--- a/core/src/net/sf/openrocket/aerodynamics/BarrowmanCalculator.java
+++ b/core/src/net/sf/openrocket/aerodynamics/BarrowmanCalculator.java
@@ -852,8 +852,7 @@ public class BarrowmanCalculator extends AbstractAerodynamicCalculator {
 	private void buildCalcMap(FlightConfiguration configuration) {
 		calcMap = new HashMap<>();
 
-		// because this is not a per-instance iteration... this usage of 'getActiveComponents' is probably fine.
-		for (RocketComponent comp: configuration.getActiveComponents()) {
+		for (RocketComponent comp: configuration.getAllComponents()) {
 			if (!comp.isAerodynamic())
 				continue;
 

